node/services/sellerService.ts

```diff
@@ -20,6 +30,22 @@ export default {
       return prev
     }, [] as any)
 
+    const settingsFetch = await settings(ctx)
+
+    for (const seller of response) {
+      if (seller?.adyenAccount) {
+        // eslint-disable-next-line no-await-in-loop
+        const accountStatus = await ctx.clients.adyenClient.getAccountHolder(
+          seller?.adyenAccount?.accountHolderCode,
+          settingsFetch
+        )
+
+        seller.adyenAccount.status =
+          accountStatus?.accountHolderStatus.status ??
+          seller.adyenAccount.status
+      }
+    }
+
```

Comment:
I think that you should put it inside a map into response constant at line 8. Just to avoid disabling eslint.
After the reduce function, use the map function with Promise.all 
```ts
const response = await Promise.all(sellers.reduce(...).map(async (...)))
```